Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:16-alpine
+
+WORKDIR /app
+
+COPY package.json package-lock.json ./
+
+RUN NODE_ENV=development npm i
+
+COPY . .
+
+RUN npm run build
```

Comment:
I believe this run command is useless, right? Because the container runs on dev mode anyway. No build files are used. Can it be removed?